providers/base/bin/wifi_nmcli_test.py

```diff
@@ -89,13 +89,10 @@ def turn_down_nm_connections():
     for name, value in connections.items():
         uuid = value["uuid"]
         print("Turn down connection", name)
-        try:
-            cmd = "nmcli c down {}".format(uuid)
-            print_cmd(cmd)
-            sp.call(shlex.split(cmd))
-            print("{} {} is down now".format(name, uuid))
-        except sp.CalledProcessError as e:
-            print("Can't down {}: {}".format(uuid, str(e)))
+        cmd = "nmcli c down {}".format(uuid)
+        print_cmd(cmd)
+        sp.run(shlex.split(cmd), check=True)
```

Comment:
Minor: slightly easier to read
```suggestion
        sp.check_call(shlex.split(cmd))
```